### ISSUE TITLE
Setting Weekdays - Clarification needed

### ISSUE THREAD

**Issue by APraxx** [zero-udo/eurotronic-cometblue#7]
Hi,

I wanted to modify my Eurotronic CometBlue. 

So i built something like 
```
blue.set_weekday(Weekday.MONDAY, 
        {"start1": "06:30", "end1": "08:00", 
         "start2": "15:30", "end2": "21:30"})
```

but when trying to send it will set it start2/end2 to 0. So not working for me
After looking into it i found in [__from_time_string](https://github.com/zero-udo/eurotronic-cometblue/blob/master/cometblue/__init__.py#L129) that the values are checked after modifying them eg.: 21:30 isn't valid because 21*6 is not in range(0, 24) .

Is this a bug or am i missing something?

Another Question arose how do you set it to OFF? The "official app" sets a magic value like 255 (42:30 in HH:mm) or is 00:00 off?   
Another thing i noticed was that contrary to your example i had to write the whole dict in set_weekday (start/end for 1,2,3,4).

Anyway thanks for writing this

**Comment by zero-udo** [comment]
Hey,

indeed this is a bug nobody noticed until now 👍 The range happens after the conversion to the eurotronics format - therefore the validity check for valid human-readable hours/minutes doesn't make sense at this point. I moved it to correct this. This one also (among other stuff) caused an issue with only setting "partial" schedules. Fixed this one as well.

For turning it off, I set start and end to the same time. This was working for me, but introducing the magic value might also be possible. Which thermostat are you using (to prevent that there are some slight deviations between models)? For me 224 (37:20 in HH:mm) is written.

I will publish new packages to pypi this evening.
